README.md

```diff
@@ -1,2 +1,270 @@
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|status|[ObservationStatus](https://hl7.org/fhir/R4B/valueset-observation-status.html)|e.g. "final"|This value is most likely "final" in every case.|
+|code|[Code](https://hl7.org/fhir/R4B/valueset-observation-codes.html)|e.g. `{"system":"http://loinc.org","code":"55423-8","display":"Number of steps"}`|Use one of the codes mentioned below.|
+|value|optional [Quantity](https://hl7.org/fhir/R4B/datatypes.html#Quantity)|e.g. `{"code":"mm[Hg]","system":"http://unitsofmeasure.org","unit":"mmHg","value":120}`|Use one of the units listed below.|
+|component|list of components|-|Instead of containing a single `value` property, some observations are composed of multiple components (e.g. a blood pressure observation contains a diastolic and systolic component).|
+|component[x]>code|[Code](https://hl7.org/fhir/R4B/valueset-observation-codes.html)|e.g. `{"system":"http://loinc.org","code":"55423-8","display":"Number of steps"}`|Use one of the codes mentioned below.|
+|component[x]>value|optional [Quantity](https://hl7.org/fhir/R4B/datatypes.html#Quantity)|e.g. `{"code":"mm[Hg]","system":"http://unitsofmeasure.org","unit":"mmHg","value":120}`|Use one of the units listed below.|
+|effective|[Period](https://fhir-ru.github.io/datatypes.html#Period)|-|For observations that happened in only one instant, we use periods with the same start and end time.|
+
+TBD: The fixed use of the period type in the `effective` property originates in HealthConnectOnFHIR. Should we also allow DateTime for instant observations?
+
+#### Compound Observations
+
+|code>system|code>code|code>display|value|components|
+|-|-|-|-|-|-|
+|"http://loinc.org"|"85354-9"|"Blood pressure panel with all children optional"|-|"Diastolic blood pressure", "Systolic blood pressure"|
+
+#### Simple Observations
+
+|code>system|code>code|code>display|value>value|value>code|value>unit|
+|-|-|-|-|-|-|
+|"http://loinc.org"|"55423-8"|"Number of steps"|double|?|"steps"|
+|"http://loinc.org"|"91557-9"|"Lean body weight"|double|"[lb_av]"|"g" or "lbs"|
+|"http://loinc.org"|"29463-7"|"Body weight"|double|"[lb_av]"|"g" or "lbs"|
+|"http://loinc.org"|"41981-2"|"Calories burned"|double|?|"kcal"|
+|"http://loinc.org"|"8302-2"|"Body height"|double|?|"m"|
+|"http://loinc.org"|"8867-4"|"Heart rate"|double|?|"beats/minute"|
+|"http://loinc.org"|"8310-5"|"Body temperature"|double|?|"°C"|
+|"http://loinc.org"|"8462-4"|"Diastolic blood pressure"|double|"mm[Hg]"|"mmHg"|
+|"http://loinc.org"|"8480-6"|"Systolic blood pressure"|double|"mm[Hg]"|"mmHg"|
+
+TODO: Currently, HealthKitOnFHIR and HealthConnectOnFHIR still use different units to encode the same observations. We should probably aim to always use the same unit to reduce unit conversions, but each client would need to be able to handle multiple units for each observation type. We should probably aim for SI units, even though the system will primarily be used in the US. TBD.
```

Comment:
Agree; let's aim that the clients do an SI unit conversion before storing the values. We should nevertheless support an on-the-fly conversion in all parts of the system (and will most likely need it for both the mobile apps) as we might also ingest some external data in the future.

Reply:
I agree. Depending on how we would integrate external data, we could also handle the conversion there (e.g. using some middleware component / firebase functions).

Reply:
I specified the respective units to be used and I think they have already been addressed in https://github.com/StanfordSpezi/SpeziKt/pull/15 before merging.